packages/web-components/package.json

```diff
@@ -94,8 +94,8 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
     "@microsoft/fast-components-styles-msft": "^4.29.0",
-    "@microsoft/fast-element": "^0.16.0",
-    "@microsoft/fast-foundation": "^1.7.0",
+    "@microsoft/fast-element": "^0.17.0",
+    "@microsoft/fast-foundation": "^1.7.1",
```

Comment:
Did you run yarn after this update? I'd expect to see an update to the yarn lockfile here.

Reply:
I did, but looks like I forgot to commit the lockfile!